test/scripts/create_vm_libvirt.sh

```
@@ -62,15 +63,16 @@ echo "Copying ${DISK_PATH_SRC} to ${DISK_PATH}..."
 cp ${DISK_PATH_SRC} ${DISK_PATH}
 
 # Resize the VM image
-qemu-img resize ${DISK_PATH} +20G  # bumping for the increased number of agent-images to be saved
+qemu-img resize ${DISK_PATH} +${VM_DISK_SIZE_INC}G  \ # bumping for the increased number of agent-images to be saved
+&& qemu-img info --output=json "${DISK_PATH}"
```

Shellcheck (0.10.0)

[error] Parsing stopped here. Mismatched keywords or invalid parentheses?

(SC1070)